tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1218,21 +1188,27 @@ module.exports = {
 
   //note to CHW
   getNoteToCHW: () => {
-    return element(by.name('/delivery/group_note/g_chw_sms')).getText();
+    return element(by.css('textarea')).getAttribute('value');
   },
 
   //summary page
   getOutcomeText: () => {
     return element(by.css('[data-value=" /delivery/group_delivery_summary/display_delivery_outcome "]'))
-      .getText();
+      .getInnerHtml();
   },
 
   getDeliveryLocationSummaryText: () => {
     return element(by.css('[data-value=" /delivery/group_summary/r_delivery_location "]'))
-      .getText();
+      .getInnerHtml();
   },
 
   getFollowUpMessage: () => {
-    return element(by.css('[data-value=" /delivery/group_note/g_chw_sms "]')).getText();
+    return element(by.css('[data-value=" /delivery/group_note/g_chw_sms "]')).getInnerHtml();
+  },
+
+  deleteReport: (done) => {
+    element(by.css('li.read.selected')).getAttribute('data-record-id').then(function(docId) {
+      utils.deleteDoc(docId).then(done).catch(done);
```

Comment:
Again, instead of `.then(done).catch(done);`, you should be able to return the actual promise from the test, which will leave you with:
```
deleteReport: () => element(by.css('li.read.selected'))
    .getAttribute('data-record-id')
    .then(docId => utils.deleteDoc(docId));
```